docs/book/themes/themes.rst

```diff
@@ -0,0 +1,105 @@
+.. index::
+   single: Themes
+
+Themes
+======
+
+Theming is a method of customizing how your channels look like in Sylius. Each channel can have a different tqheme.
```

Comment:
I'd add more information about why using themes. From my understanding:
 - use standard Symfony overriding technique if:
 -- you have only one channel
 -- you don't need multiple themes at the same time configured
 -- you want something basic (e.g. change the color set of the website or re-arrange some  blocks)

 - use Sylius themes overriding technique if:
 -- you have more than one channel enabled for a single Sylius instance and every channel should have their own look and behavior
 -- etc. etc.